test_conformance/api/procs.h

```diff
@@ -89,6 +89,8 @@ extern int        test_min_max_language_version(cl_device_id deviceID, cl_contex
 extern int        test_native_kernel(cl_device_id device, cl_context context, cl_command_queue queue, int n_elems );
 
 extern int      test_create_context_from_type(cl_device_id deviceID, cl_context context, cl_command_queue queue, int num_elements);
+extern int      test_create_context_from_type_device_type_all(cl_device_id deviceID, cl_context context, cl_command_queue queue, int num_elements);
+extern int      test_create_context_from_type_device_type_default(cl_device_id deviceID, cl_context context, cl_command_queue queue, int num_elements);
```

Comment:
context_from_type goes beyond just creating the context by creating memory objects and enqueueing a kernel etc.
Should these other tests do that as welll?
num_elements seems to be unused in all cases and could be removed.

Reply:
@lakshmih No, I just wanted to create test scenarios with clCreateContextFromType with TYPE_ALL and TYPE_DEFAULT. I did not find any sufficient place so I put it here.
num_elements will be removed.